README.md

```diff
@@ -28,7 +30,21 @@ Launch the REPL
 ./aerepl
 ```
 
-* Basic usage
+## Docker image
+
+For a consistent setup, a docker image can be created:
+
+```
+make docker
+```
+
+Then to start the dockerized REPL:
+
+```
+docker run -i aeternity/aerepl
```

Comment:
make docker produces `aeternity/aerepl:local` so you'd probably want that :local tag in this cmd as well

Reply:
right